specification/azurestackhci/AzureStackHCI.StackHCIVM.Management/main.tsp

```diff
@@ -27,6 +25,7 @@ using TypeSpec.Versioning;
  * Azure Stack HCI management service
  */
 #suppress "@azure-tools/typespec-azure-core/casing-style" "HCI itself is a short form."
+#suppress "@azure-tools/typespec-azure-resource-manager/missing-operations-endpoint"
```

Comment:
Please indicate the reason for this suppression, so it's clear to future readers.

I'm approving this suppression, since the operations API is still spec'd, but now resides in another spec in this namespace.